README

```diff
@@ -40,8 +40,9 @@ GNU versions of these tools are recommended.
 
 Installation
 ------------
-Installing Geany is done by the following three commands:
-$ ./configure
+Installing Geany is done by the following commands:
+$ libtoolize
```

Comment:
Why have put this explicit here? In about most cases its not needed from my experince